exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillJdbc41Factory.java

```diff
@@ -103,9 +103,9 @@ public DrillResultSetImpl newResultSet(AvaticaStatement statement,
                                          TimeZone timeZone) {
     final ResultSetMetaData metaData =
         newResultSetMetaData(statement, prepareResult.getColumnList());
-    return new DrillResultSetImpl( (DrillStatementImpl) statement,
-                                   (DrillPrepareResult) prepareResult,
-                                   metaData, timeZone);
+    return new DrillResultSetImpl(statement,
+                                  (DrillPrepareResult) prepareResult,
```

Comment:
Cast here is redundant here, DrillResultSetImpl() is expecting an AvaticaPrepareResult object